source/blog/2019-02-14-octane-guides.md

```diff
@@ -0,0 +1,59 @@
+
+## Updating the guides
+
+As per the usual standard, we have planned out this work to be able to complete it in an iterative fashion. This was particularly important because we are updating guides as new features for the Octane release are merged into Ember. 
+
+### Phases
+
+This is roughly the planned phases of the documentation work.
+
+Phase 1, Octane Guides, MVP: (see [Tracking issue](https://github.com/ember-learn/guides-source/issues/394))
+
+1. Create octane branch of the guides for this work
+1. Reorder the table of contents (as per the RFC)
+1. Add in placeholders for any new sub-sections
+1. Add or update content in each sub-section (prose, code samples) [https://github.com/ember-learn/guides-source/issues/394](https://github.com/ember-learn/guides-source/issues/394)
+1. Review, Edit & re-arrange sub-sections before preview launches
+1. Add re-direct URLs and remove old content
+
+Then the Octane guides will be considered in MVP status and we will move forward to the next phase, which will include all of the sections that were identified in the table of contents RFC but were not included in the MVP. 
+
+Through all of these phases, a preview of the Octane edition guides will be used to ensure that the documentation is staying on track with the intended features. 
+
+While new content is being written and older content is being updated, **no existing content will be completely deleted.** Only _after_ we have fully reviewed the content and added the re-directs will outdated content be removed. This means that in the interim, some content may be available if the user has the URL, but it won't be linked from the table of contents. 
+
+### Challenges
+As with any ambitious undertaking, there are some related challenges. Here are a few we are currently facing: 
+
+- converting guides from classic syntax to new syntax
+- the number of sub-sections for each guide section
+- handling redirects for old content in a way that won't break deprecation messages
+
+We expect to work through these challenges and have process in place to handle them, but we should have a shared awareness that our approach may be tweaked as the next month goes on, to address these issues. 
+
+## How you can help
+We expect that the community will be willing to submit PRs to help make the documentation better- from checking for spelling or grammar errors, to adding code samples, and even more ambitiously, writing some prose. 
+
+Thanks to our infrastructure upgrades in 2018, we are well-positioned to accept community participation in the documentation effort. Please see the open issues in the [guides source repository](https://github.com/ember-learn/guides-source/issues) and let us know in the #dev-ember-learning channel (on Discord) if you would like to work on an issue. 
```

Comment:
Shouldn't we tell people to post in the issues to claim ownership? That way if someone else comes in, they can see there are people signed up to it.

Reply:
Typically we do, but this time we want interaction immediately, so we can provide some direction, etc.